### DISCUSSION TITLE
Copy password by clicking inside the password box.

### DISCUSSION THREAD

**Discussion by andyplank** [bitwarden/clients#6404] [Password Manager]
### Code Contribution Proposal

I want to add a new way to copy your password on Windows desktop. Right now, you have to click the copy password button on the very right side of the screen. I want to also allow the user to copy their password by clicking anywhere inside the password box. For example, if the user clicked on the red dot marked in the picture it would copy the password to their clipboard.
![image](https://github.com/bitwarden/clients/assets/32016647/227caa4d-69ea-4e53-84e7-d9575f346c8b)

**Comment by dwbit** [comment]
Closing out this older request, feel free to ping if you need anything.
